guides/doc-Administering_Red_Hat_Satellite/topics/Security_Compliance_Management.adoc

```diff
@@ -5,6 +5,11 @@ Security compliance management is the ongoing process of defining security polic
 Any non-compliance is managed according to the organization's configuration management policies.
 Security policies range in scope from host-specific to industry-wide, therefore, flexibility in their definition is required.
 
+ifndef::satellite[]
+Note that the following chapter refers only to RPM-based systems.
+We welcome any and all equivalent documentation for Debian users.
+endif::[]
+
 [[sect-Administering-OpenSCAP_Concepts]]
```

Comment:
Doesn't this work? It probably needs to be done a layer above, but you get the idea.
```suggestion
ifndef::foreman-deb[]
[[sect-Administering-OpenSCAP_Concepts]]
endif::[]
```